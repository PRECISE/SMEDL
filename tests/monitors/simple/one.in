{
    "fmt_version": [2, 0],
    "channel": "conn",
    "event": "in",
    "params": [2],
    "aux": {
        "arbitrary": "json data"
     }
} 
