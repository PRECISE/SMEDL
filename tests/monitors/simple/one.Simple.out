_Simple_out,2
